File: Models/BusinessSetUp.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace MtpApp.Models
{
    public class BusinessSetUp
    {
        public int Id { get; set; }
        [Required]
        public int ClientId { get; set; }
        public Client Client { get; set; }
        [Required]
        public int BusinessSetUpCategoryId { get; set; }
        public BusinessSetUpCategory BusinessSetUpCategory { get; set; }
        [Required]
        [StringLength(255)]
        public string BusinessType { get; set; }
        [Required]
        [StringLength(255)]
        public string Income { get; set; }
        [Required]
        [StringLength(255)]
        public string Expense { get; set; }
        [Required]
        [StringLength(255)]
        public string CountTime { get; set; }
        public string JobPlaceBy { get; set; }
        public DateTime? StartBusinessSetUpDate { get; set; }
  
    }
}

File: spring-backend/src/main/java/com/mtp/api/models/BusinessSetUp.java
package com.mtp.api.models;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import java.time.LocalDateTime;

@Entity
@Table(name = "BusinessSetUps")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class BusinessSetUp {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "ClientId")
    private Client client;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "BusinessSetUpCategoryId")
    private BusinessSetUpCategory businessSetUpCategory;

    @NotNull
    @Size(max = 255)
    private String businessType;

    @NotNull
    @Size(max = 255)
    private String income;

    @NotNull
    @Size(max = 255)
    private String expense;

    @NotNull
    @Size(max = 255)
    private String countTime;

    private String jobPlaceBy;

    private LocalDateTime startBusinessSetUpDate;
}
